src/java/org/apache/cassandra/cql3/statements/schema/AlterTableStatement.java

```diff
@@ -767,6 +839,8 @@ public AlterTableStatement prepare(ClientState state)
                 case        RENAME_COLUMNS: return new RenameColumns(keyspaceName, tableName, renamedColumns, ifTableExists, ifColumnExists);
                 case         ALTER_OPTIONS: return new AlterOptions(keyspaceName, tableName, attrs, ifTableExists);
                 case  DROP_COMPACT_STORAGE: return new DropCompactStorage(keyspaceName, tableName, ifTableExists);
+                case       DROP_CONSTRAINT: return new DropConstraint(keyspaceName, tableName, ifTableExists, constraintName);
+                case       ALTER_CONSTRAINTS: return new AlterConstraints(keyspaceName, tableName, ifTableExists, constraintName, constraints);
```

Comment:
@bbotella please align all other cases with ALTER_CONSTRAINTS so `:` is on the same position